### PR TITLE
Update Compute SDK Package

### PR DESCRIPTION
**Pull request by grizzlytheodore**
## Description
updating Compute .net SDK package to the newest.
Needed to update New-RestorePoint to follow the newest SDK parameters, but no breaking change in customer's side

## Checklist

- [x] I have read the [_Submitting Changes_](../blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/master/CONTRIBUTING.md)
- [x] The title of the PR is clear and informative
- [x] The appropriate `ChangeLog.md` file(s) has been updated:
    - For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header -- no new version header should be added
- [x] The PR does not introduce [breaking changes](../blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [ ] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (_Microsoft internal only_)
        - {Please put the link here}
    - [ ] the markdown help files have been regenerated using the commands listed [here](../blob/master/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)

### CODE REVIEW

File: src/Compute/Compute/Generated/Models/PSRestorePoint.cs [hunk 1]
```diff
@@ -35,10 +35,7 @@ class PSRestorePoint
         //     for more details. Possible values include: 'CrashConsistent', 'FileSystemConsistent',
         //     'ApplicationConsistent'
         public string ConsistencyMode { get; set; }
-        //
-        // Summary:
-        //     Gets the provisioning details set by the server during Create restore point operation.
-        public RestorePointProvisioningDetails ProvisioningDetails { get; set; }
```
Comment (wyunchi-ms) [inline src/Compute/Compute/Generated/Models/PSRestorePoint.cs:41]: Removing property in output type will cause breaking change. We cannot accept this PR unless major release.

**Comment by grizzlytheodore** [comment]
@wyunchi-ms please review /merge

**Comment by grizzlytheodore** [comment]
I am not sure why the tests are failing the way they are.  
This PR does not make any changes to tests that are failing also the the matching HTTP requests can be found. 

For example one of the test failure log goes:
Microsoft.Azure.Commands.Compute.Test.ScenarioTests.VirtualMachineScaleSetTests.TestVirtualMachineScaleSetLB [FAIL]
2022-02-10T02:10:01.6030323Z     X Microsoft.Azure.Commands.Compute.Test.ScenarioTests.VirtualMachineScaleSetTests.TestVirtualMachineScaleSetLB [2s 127ms]
2022-02-10T02:10:01.6076239Z EXEC : error Message:  [D:\a\1\s\build.proj]
2022-02-10T02:10:01.6085734Z      System.Management.Automation.ActionPreferenceStopException : The running command stopped because the preference variable "ErrorActionPreference" or common parameter is set to Stop: Unable to find a matching HTTP request for URL 'GET /subscriptions/24fb23e3-6ba3-41f0-9b6e-e41131d5d61e/resourceGroups/crptestps4612/providers/Microsoft.Compute/virtualMachineScaleSets/vmsscrptestps4612/virtualMachines/0?'. Calling method Item().

However that can be found here: https://github.com/Azure/azure-powershell/blob/feature/cplat-updateSDK/src/Compute/Compute.Test/SessionRecords/Microsoft.Azure.Commands.Compute.Test.ScenarioTests.VirtualMachineScaleSetTests/TestVirtualMachineScaleSetLB.json#L2867

**Comment by grizzlytheodore** [comment]
could somebody help? @wyunchi-ms

**Comment by grizzlytheodore** [comment]
@VeryEarly  please review/merge

**Comment by grizzlytheodore** [comment]
@ArcturusZhang  can you review/merge?

**Review by VeryEarly — APPROVED** [review]

**Review by VeryEarly — APPROVED** [review]

**Pull request merged**